### PR TITLE
Embed page + Population detail on profile detail component

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

Embed:
```
<iframe src="http://localhost:3000/embed/country-SN/elections/valid-vs-invalid-votes/" />
```

Screen Shot 2019-09-09 at 1 21 58 PM

Fixes # (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by karimkawambwa** [comment]
@KhadijaMahanga get #8 done and merged so I can update this PR.

File: src/pages/Embed.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+  return (
+    <div style={{ maxWidth: '100%' }}>
+      <InsightContainer
+        key={chart.id}
+        loading={chartData.isLoading}
+        title={chart.title}
+        source={{
+          title: 'Community Survey 2016',
+          href: 'http://dev.dominion.africa'
```
[the remark this answers is not in the document]
Comment (kilemensi) [reply src/pages/Embed.js]: `master` has sources @karimkawambwa: https://github.com/TakwimuAfrica/Takwimu.AFRICA/blob/master/src/data/sources.json

**Comment by kilemensi** [comment]
> @kilemensi
> 
> > Any way to handle connecting to locally running Takwimu (http://localhost:8000) during development?
> 
> Set the proxy to `http://localhost:8000`

Yeah that would work @karimkawambwa ... but that means I have to edit both `package.json` _and_ `setupProxy.js`.

Poor me was just hoping for a simpler approach... environment variables or something. But no biggie.

File: src/setupProxy.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const axios = require('axios');
+
+module.exports = app => {
+  app.use('/flourish/:id', async (req, res) => {
+    axios
+      .get(`https://takwimu.africa/flourish/${req.params.id}`)
```
[the remark this answers is not in the document]
Comment (karimkawambwa) [reply src/setupProxy.js]: Ohh wait @kilemensi, isn't cross-fetch a browser thing. This is server side so i believe i should be using axios.

[the remark this answers is not in the document]
Comment (kilemensi) [reply src/setupProxy.js]: Yeah, `fetch` is browser only (or with `node-fetch` package)... Cool, lets move `axios` to `devDependencies` then.

**Comment by karimkawambwa** [comment]
@kilemensi ahh i get you, i'll keep track of this and probably introduce an env variable. or maybe proxy can be accessed from `app.get('proxy')`.

**Pull request merged**